docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-portal-product-management-feature-walkthrough.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Marketplace Merchant Portal Product Management feature walkthrough
+description: This article provides reference information about product in the Merchant Portal.
+template: concept-topic-template
+---
+
+The *Marketplace Merchant Portal Product Management* feature allows merchants to manage product in the Merchant portal.
+
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/a38bb45f-0f1c-4153-8f3d-7873b3aa13af.png?utm_medium=live&utm_source=custom)
```

Comment:
Please add the main modules that ProductMerchantPortalGui depends on. (Please use white color for them)